src/cms/templates/feedback/feedback_form.html

```diff
@@ -0,0 +1,106 @@
+{% extends "_base.html" %}
+{% load i18n %}
+{% block content %}
+{% load static %}
+{% load widget_tweaks %}
+<h2>Feedback</h2> 
+
+<table class="w-full mt-4 rounded border border-solid border-gray-200 shadow bg-white" border="1">
+<tr class="border-b border-solid border-gray-200">
+  <th>Type</th>
+  <th>Objekt</th>
+  <th>Comment</th>
+  <th>Rating</th>
+  <th>Date</th>
+</tr>
+
+
+{% for item in feedback_events %} 
+<tr><td align="center">Event</td>
+<td align="center">https://integreat.app/{{item.event.permalink}}</td>
+<td align="center">{{ item.comment }}</td>
+  {% if item.emotion == "NEG"  %}
+       <td align="center"> Down </td>
+  {% else %}
+      <td align="center" >Up</td>
+  {% endif %}
+  <td align="center">{{ item.created_date }}</td></tr>
+{% endfor %}
+{% for item in feedback_offers %}
+<tr><td align="center">Offer</td>
+<td align="center">{{item.offer.url}}</td>
+  <td align="center">{{ item.comment }}</td>>
+   {% if item.emotion == "NEG" %}
+       <td align="center"> Down </td>
+  {% else %}
+      <td align="center">Up</td>
+  {% endif %}<td align="center">{{ item.created_date }}</td></tr>
+{% endfor %}
+{% for item in feedback_regions %}
+<tr><td align="center">Region</td>
+  <td align="center">{{item.region.permalink}}</td>
+  <td align="center">{{ item.comment }}</td>
+  {% if item.emotion == "up"  %}
+    <td align="center"> up</td>
+  {% else %}
+     <td align="center">down</td>
+  {% endif %}<td align="center">{{ item.created_date }}</td></tr>
+{% endfor %}
+{% for item in feedback_pages %}
+<tr><td align="center">Pages</td>
+  <td align="center">{{ item.page.url }}</td>
+  <td align="center">{{ item.comment }}</td>
+  {% if item.emotion == "up"  %}
+    <td align="center"> up</td>
+  {% else %}
+    <td align="center"> down</td>
+  {% endif %}<td align="center">{{ item.created_date }}</td></tr>
+{% endfor %}
+</table>
+<br>
+<h2>Rating</h2>
+
+  <table class="w-full mt-4 rounded border border-solid border-gray-200 shadow bg-white">
+    <tr class="border-b border-solid border-gray-200">
+      <th>Type</th> 
+      <th>Like &#128077;</th>
+      <th>DisLike &#128078;</th>
+    </tr>
+  {% if feedback_events %}
+  <tr >
+    <td align="center"> Event</td>
+    <td align="center">{{countUp_event}}</td>
+    <td align="center">{{countDown_event}}</td>
+  </tr>
+  {%else%}
+  {%endif%}
+  {% if feedback_offers %}
+  <tr >
+    <td align="center">Offer</td>
+    <td align="center">{{countUp_offer}}</td>
+    <td align="center">{{countDown_offer}}</td>
+  </tr>
+  {%else%}
+  {%endif%}
+  {% if feedback_regions %}
+  <tr >
+    <td align="center">Region</td>
+    <td align="center">{{countUp_region}}</td>
+    <td align="center">{{countDown_region}}</td>
+  </tr>
+  {%else%}
+  {%endif%}
+  {% if feedback_pages %}
+  <tr >
+    <td align="center">Page</td>
+    <td align="center">{{countUp_page}}</td>
+    <td align="center">{{countDown_page}}</td>
+  </tr>
+  {%else%}
+  {%endif%}
+</table>
+  
+
+{% endblock %}
+
+
```

Comment:
Two general notes about the template:
1. Please rename it to `feedback_list.html`, since it is a list/table and not a form.
2. Please make sure it looks the same as all our other lists/tables. You can just copy the css classes of other lists and apply this to this one.